ui/selectors/accounts.ts

```diff
@@ -20,6 +19,10 @@ function isBtcAccount(account: InternalAccount) {
   return Boolean(account && account.type === P2wpkh);
 }
 
+export function getInternalAccounts(state: AccountsState) {
+  return Object.values(state.metamask.internalAccounts.accounts);
+}
```

Comment:
We decided that as we refactor, anything that is a selector should start with `select` to better follow the guidelines.  I don't know if a "Replace in files" for `getInternalAccounts` finds too many, I took a quick look and it's 100 results.

Reply:
Huh. Isn't `get*` way more prevalent than `select*`? Maybe 10x more common in our code base?

Related: switching to use `select` instead of `get*` would result in a bunch of `selectSelected*` selectors, Which is weird.

Reply:
Yes, all over our codebase we are not following the Redux guidelines
https://redux.js.org/style-guide/#name-selector-functions-as-selectthing

Reply:
If I must I will, but it's unrelated to the intent of this refactor. My intent with all of these PRs is to just move functions as necessary. There are lots of actually useful refactoring I could perform when moving these functions, but (mostly) don't because they'll complicate the reviews.

Reply:
To further clarify the scope of what you are asking here, my intent is to automate as much of this types of changes as possible (automation coming in a future PR), but I'm not planning on extending the code to automatically rename any functions.